### ISSUE TITLE
High power consumption during sleep in hybrid mode

### ISSUE THREAD

**Issue by niciuffo** [Askannz/optimus-manager#159]
**Describe the bug**
After the PC has gone to sleep, some component doesn't fully switch off and still draws power. I am assuming it's the GPU, but I'm not sure how to test this.
The 80Wh battery on my notebook goes from 100% to 50% in around 5.5 hours, which indicates a power consumption of ~7.3W. The notebook is very warm to the touch.

**System info**
OS: Arch Linux
Desktop manager: KDE Plasma 5.16.5-1
Display manager: SDDM 0.18.1-1
Laptop: Lenovo X1 Extreme (Gen 2) [supports PRIME]
optimus-manager 1.2.1-1
optimus-manager-qt 1.3.3-1

**Logs**
[boot_setup.log](https://github.com/Askannz/optimus-manager/files/3722542/boot_setup.log)
[gpu_setup.log](https://github.com/Askannz/optimus-manager/files/3722543/gpu_setup.log)
[prime_setup.log](https://github.com/Askannz/optimus-manager/files/3722544/prime_setup.log)

**optimus-manager.conf**
```
 [optimus]
auto_logout=yes
pci_power_control=no
pci_remove=no
pci_reset=no
switching=bbswitch
```

**Comment by Askannz** [comment]
Sorry, not much I can do here. This is likely a bug in the Nvidia driver, so you should report it to them. Try disabling the new power management method if you have it enabled, or try older/more recent versions of the driver.

Actually, it looks like the newly released [440.26 version](https://www.nvidia.co.uk/download/driverResults.aspx/152600/en-uk) has a bugfix that may be related to your issue:

> Fixed a bug that prevented NVIDIA GPUs from entering the low-power D3 state when entering suspend-to-idle (s2idle). Note that on some systems this may expose an existing timing issue in the Linux kernel; see the "Known Issues and Workarounds" section of the "Configuring Power Management Support" chapter of the README for more details.

**Comment by niciuffo** [comment]
This wasn't caused by optimus-manager. It looks to be a bug affecting some devices starting from kernel 5.2 - bug reports [here](https://bugzilla.redhat.com/show_bug.cgi?id=1748936) and [here](https://bugzilla.kernel.org/show_bug.cgi?id=203791). Solution seems to be to downgrade to 5.1.9 until a fix. Just posting in case someone stumbles upon this thread.
